src/code/z_message.c

```diff
@@ -4531,10 +4531,10 @@ void Message_DrawMain(PlayState* play, Gfx** gfxP) {
             case MSGMODE_OCARINA_NOTES_DROP:
             case MSGMODE_SONG_PROMPT_NOTES_DROP:
                 for (i = 0; i < 5; i++) {
-                    msgCtx->ocarinaButtonsPosY[i] += D_801C6A94;
+                    msgCtx->ocarinaButtonsPosY[i] += sOcarinaButtonDropYOffset;
                 }
-                D_801C6A94 += D_801C6A94;
-                if (D_801C6A94 >= 0x226) {
+                sOcarinaButtonDropYOffset += sOcarinaButtonDropYOffset;
+                if (sOcarinaButtonDropYOffset >= 0x226) {
```

Comment:
```suggestion
                if (sOcarinaButtonDropYOffset >= 550) {
```
Since it seems to be a position offset I think decimal is better here.